include/gauge_field_order.h

```diff
@@ -2998,6 +3010,10 @@ namespace quda {
     typedef gauge::FloatNOrder<char, N, 4, 12, stag, huge_alloc, ghostExchange, use_inphase> type;
   };
   template <int N, QudaStaggeredPhase stag, bool huge_alloc, QudaGhostExchange ghostExchange, bool use_inphase>
+  struct gauge_mapper<char, QUDA_RECONSTRUCT_10, N, stag, huge_alloc, ghostExchange, use_inphase> {
+    typedef gauge::FloatNOrder<char, N, 2, 10, stag, huge_alloc, ghostExchange, use_inphase> type;
```

Comment:
Is that supposed do `<char, N, 2, 10, stag ...>` here? Just looking at the diff and above QUDA_RECONSTRUCT_10 is using 11.

Reply:
Good catch, this is a bug.  It should be 11 here too.  Will fix this in my next push.

This is something that will eventually be cleaned up, since the `11` is unintuitive (it comes from prior ignorance of C++).